components/list/list-item-placement-marker.js

```diff
@@ -0,0 +1,91 @@
+import '../colors/colors.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class ListItemPlacementMarker extends RtlMixin(LitElement) {
+
+	static get styles() {
+		return css`
+			:host {
+				display: block;
+			}
+
+			:host([hidden]) {
+				display: none;
+			}
+
+			.d2l-list-drag-marker-line {
+				height: 12px;
+				width: 100%;
+				stroke: var(--d2l-color-celestine);
+				stroke-width: 3px;
+				margin-left: -1px;
+				margin-right: -1px;
+			}
+
+			.d2l-list-drag-marker-linecap {
+				height: 12px;
+				width: 4px;
+				stroke: none;
+				fill: var(--d2l-color-celestine);
+				margin-left: -1px;
+				margin-right: 0px;
```

Comment:
Nit: `0px` should just be `0`.  It used to be a micro-optimization for performance, but nowadays I'd say it's just a consistency thing.